cli/azd/pkg/tools/swa/swa.go

```diff
@@ -17,7 +17,7 @@ import (
 )
 
 // cSwaCliPackage is the npm package (including the version version) we execute with npx to run the SWA CLI.
-const cSwaCliPackage = "@azure/static-web-apps-cli@1.1.8"
+const cSwaCliPackage = "@azure/static-web-apps-cli@2.0.1"
```

Comment:
Would this be considered a breaking change to anyone using old 1.x?

Reply:
The only breaking change is that 2.0 requires node18.
Azd requires node18 since about a year ago, so, I don't think anyone would be affected